src/main/java/com/browserstack/pages/Bag.java

```diff
@@ -4,19 +4,64 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.By;
 
 public class Bag {
 
 	WebDriver driver;
 
 	public final static String checkoutselector = "#__next > div > div > div.float-cart.float-cart--open > div.float-cart__content > div.float-cart__footer > div.buy-btn";
+	public final static String SignInSelector = "signin";
+	public final static String iPhone12Locator = "#\\31  > div.shelf-item__buy-btn";
+	public final static String iPhone12MiniLocator = "#\\32  > div.shelf-item__buy-btn";
+	public final static String iPhone12ProMaxLocator = "#\\33  > div.shelf-item__buy-btn";
+	public final static String OrderButtonLocator = "orders";
 
-	public Bag(WebDriver driver) {
+
+	public Bag(WebDriver driver, WebDriverWait wait) {
 		this.driver = driver;
 		PageFactory.initElements(driver, this);
+		AddToCart(wait);
+	}
+
+	public void AddToCart(WebDriverWait wait) {
```

Comment:
Method names must begin with a lower case letter.

Reply:
Done, renamed all methods that were starting with uppercase letter